proto/xudrpc.proto

```diff
@@ -145,6 +145,14 @@ service Xud {
   }
 
 
+  /* Execute a swap between a maker peer order and a taker own order */
```

Comment:
I think with the new approach, this should just say "Execute a swap on a maker peer order" or something like that.